src/main/java/net/javadiscord/javabot/systems/help/HelpListener.java

```diff
@@ -206,44 +212,33 @@ private void handleHelpThanksInteraction(@NotNull ButtonInteractionEvent event,
 			return;
 		}
 		switch (id[2]) {
-			case "done" -> handleThanksCloseButton(event, manager, post, config);
+			case "done" -> handleThanksCloseButton(event, manager, post);
 			case "cancel" -> event.getMessage().delete().queue();
 			default -> {
-				List<Button> thankButtons = event.getMessage().getButtons().stream()
-						.filter(b -> b.getId() != null &&
-								!ComponentIdBuilder.split(b.getId())[2].equals("done") &&
-								!ComponentIdBuilder.split(b.getId())[2].equals("cancel")
-						).toList();
+				List<Button> thankButtons = event.getMessage().getButtons().stream().filter(b -> b.getId() != null && !ComponentIdBuilder.split(b.getId())[2].equals("done") && !ComponentIdBuilder.split(b.getId())[2].equals("cancel")).toList();
```

Comment:
This is significantly less readable that the previous code.

Reply:
autoformat. will update my preferences.